index.js

```diff
@@ -42,6 +42,11 @@ module.exports = {
     "no-dupe-keys": 2,
     "no-underscore-dangle": 0,
     "no-undef": 2,
+    "max-lines": ["error", {
+      "max": 300,
+      "skipBlankLines": true,
+      "skipComments": true
```

Comment:
Was this what we settled on? Or it just seemed good to skip blank lines and comments? I would include both of these, but I might be the minority 😄

Reply:
I feel like these were the defaults... going off the list of files that were provided in the Trello story (I think it was more than I commented), it should probably be removed though.

Reply:
I was wrong, thanks for pointing that out!